htdocs/societe/class/societe.class.php

```diff
@@ -873,6 +873,11 @@ class Societe extends CommonObject
 	 */
 	public $bank_account;
 
+	/**
+	 * @var int
+	 */
+	public $no_button_copy;
```

Comment:
This property is no more used in develop. Can you remove  it?

Reply:
@eldy I just did it.